lib/std/sums.nim

```diff
@@ -6,12 +6,16 @@
 #    See the file "copying.txt", included in this
 #    distribution, for details about the copyright.
 
-## Fast sumation functions.
+## Fast summation functions.
```

Comment:
how about:
```suggestion
## Accurate summation functions.
```

"fast" is meaningless, you might as well just call `for ai in a: result.add ai` and it'd be faster